spec/spec_helper.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
```

Comment:
Let's remove all the unnecessary comments from all the files

Reply:
removed apu